src/main/resources/application.yaml

```diff
@@ -77,3 +77,7 @@ resilience4j:
           limitForPeriod: 15
           limitRefreshPeriod: 1s
           timeoutDuration: 0
+logging:
```

Comment:
Can we fix the issues instead of suppressing? 

If not, its probably better we still show them so we are made aware of any issues

Reply:
fixed as many warning as I can on please can you review again